src/napari_live_recording/ui/__init__.py

```diff
@@ -83,7 +85,10 @@ def deleteCameraUI(self, cameraKey: str) -> None:
         self.mainController.deleteCamera(cameraKey)
         self.mainLayout.removeRow(self.cameraWidgetGroups[cameraKey])
         del self.cameraWidgetGroups[cameraKey]
-    
+
+    def _enableDeleteButton(self, status: bool):
+        self.deleteButton.setEnabled(not status)
+
```

Comment:
Please remove this. I will test on my side and see what happens; with the latest changes I added it should work now